src/ADTSReader.cpp

```diff
@@ -132,8 +132,7 @@ bool ADTSFrame::parse(AP4_ByteStream *stream)
 /**********************************************************************************************************************************/
 
 
-ADTSReader::ADTSReader(AP4_ByteStream *stream)
-  : m_stream(stream)
+ADTSReader::ADTSReader(AP4_ByteStream* stream) : m_stream{stream}, m_basePts{0}
```

Comment:
nice small fixes are the most obscure
but instead add here, please initialize vars by using his class source
https://github.com/xbmc/inputstream.adaptive/blob/72a4cc3610574b908738dc84c204ab4bc8400e66/src/ADTSReader.h#L89

Reply:
Done :)